internal/agentsession/capture_test.go

```
@@ -25,19 +28,59 @@ func writeHermesStore(t *testing.T, rows ...hermesRow) string {
+		timestamp REAL NOT NULL
 	)`); err != nil {
 		t.Fatal(err)
 	}
 	for _, row := range rows {
-		if _, err := db.Exec(`INSERT INTO sessions (id, source, cwd, started_at) VALUES (?, ?, ?, ?)`,
-			row.id, row.source, row.cwd, float64(row.started.UnixNano())/float64(time.Second)); err != nil {
+		var activity any
+		if !row.activity.IsZero() {
+			activity = float64(row.activity.UnixNano()) / float64(time.Second)
+		}
+		if _, err := db.Exec(`INSERT INTO sessions (id, source, cwd, started_at, last_activity_at) VALUES (?, ?, ?, ?, ?)`,
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)